ParseUI/ParseUI.xcodeproj/project.pbxproj

```diff
@@ -2162,8 +2166,10 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MARKETING_VERSION = 1.18.0;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
The rest of the project files say NO to Catalyst support. I assume this is a mistake?